lib/capybara/node/finders.rb

```diff
@@ -78,57 +78,59 @@ def ancestor(*args, **options, &optional_filter_block)
 
       ##
       #
-      # Find an {Capybara::Node::Element} based on the given arguments that is also a sibling of the element called on. +sibling+ will raise an error if the element
+      # Find an {Capybara::Node::Element} based on the given arguments that is also a sibling of the element called on. {#sibling} will raise an error if the element
       # is not found.
       #
-      #
-      # +sibling+ takes the same options as +find+.
+      # {#sibling} takes the same options as {#find}.
       #
       #     element.sibling('#foo').find('.bar')
       #     element.sibling(:xpath, './/div[contains(., "bar")]')
       #     element.sibling('ul', text: 'Quox').click_link('Delete')
       #
-      # @param (see Capybara::Node::Finders#find)
-      #
       # @macro waiting_behavior
       #
-      # @option options [Symbol] match        The matching strategy to use.
-      #
-      # @return [Capybara::Node::Element]      The found element
-      # @raise  [Capybara::ElementNotFound]    If the element can't be found before time expires
+      # @overload sibling([kind], locator, **options)
+      # @param (see #find)
+      # @return (see #find)
```

Comment:
why?

Reply:
I want to be DRY also here, but this is too much. 💦